py/dynesty/pool.py

```diff
@@ -93,6 +96,17 @@ class Pool:
     Also note that you have to provide the .loglike/.prior_transform attributes
     from the pool object to the Nested samper rather than your original
     functions!
+
+    If your likelihood function takes additional arguments, it is better to
+    pass them when creating the pool, rather then to nested sampler::
+
+        with dynesty.pool.Pool(16, loglike, prior_transform, 
+                                            logl_args=(...) ) as pool:
+            dns = DynamicNestedSampler(pool.loglike, pool.prior_transform, ndim,
+                                     pool=pool)
+
+    as this way they will not need to be pickled and unpickled every function
+    call.
```

Comment:
I think it'd be good to also include this info here even though it isn't strict the pool itself so much as the sampler initialization.
```suggestion
    call.
    
    Note though that if you specify logl_args, and ptform_args when  creating
    the Pool *AND* in the sampler those will be concatenated
```